### DISCUSSION TITLE
Strange Crash with 200MB logfile

### DISCUSSION THREAD

**Discussion by Rabtos** [YimMenu/YimMenu#2202] [General]
Any one experiences this too? 

![grafik](https://github.com/YimMenu/YimMenu/assets/124796345/c90a51f3-1826-444b-8c54-2167bbdbb777)

It kept going giving me errors until i closed GTA. in 10 seconds I got a 200 mb file

**Comment by lonelybud** [comment]
it may happen. I had a similar crash. I believe mine was related to pointers referencing something they should nt.

**Reply by lonelybud** [reply]
zip the log and attach it here if it seems like a bug or something
